compiler/util.py

```diff
@@ -101,6 +101,12 @@ def write_checked_call2(self, result, call, *args, **kwargs):
         \tcontinue
         }"""), result=result.name, call=call.format(*args, **kwargs))
 
+  def write_checked_call2_as_temp(self, call, *args, **kwargs):
```

Comment:
Hi, I added the `blank_expr` as an instance of  `GeneratedLiteral`, but looks like `write_checked_call2` need a instance with a name property, so I added the name property to `GeneratedLiteral`, I don't know whether it's suitable or not.